composer.json

```diff
@@ -9,27 +9,27 @@
         { "name": "Jeremy Mikola", "email": "jmikola@gmail.com" }
     ],
     "require": {
-        "php": ">=5.3.2",
-        "symfony/config": "^2.2 || ^3.0 || ^4.0",
-        "symfony/dependency-injection": "^2.2 || ^3.0 || ^4.0",
-        "symfony/http-kernel": "^2.2 || ^3.0 || ^4.0",
-        "symfony/security-bundle": "^2.2 || ^3.0 || ^4.0",
-        "jmikola/auto-login": "^1.2.3"
+        "php": ">=7.1",
+        "symfony/config": "^4.3",
```

Comment:
@SvetlinStaev: I realized this PR and jmikola/AutoLogin#21 were opened before 5.0 was released, but should I amend these with `|| ^5.0` before making a release? I assume 4.3.0 had all of the necessary deprecations for 5.0, but I wasn't able to find a definitive answer on that. I'm basing this assumption on my read of the [UPGRADE-4.4.md](https://github.com/symfony/symfony/blob/master/UPGRADE-4.4.md) document, which appears to be irrelevant to anything you updated in these PRs.

Reply:
I believe it is compatible as 4.4 = 5.0 - deprecations.